src/browserClip.ts

```diff
@@ -111,13 +111,11 @@ const recursivelyApplyPatternToDurations = (
   return durations;
 };
 
-/**
- * @param  {Object}
- * @return {Tone.js Sequence Object}
```

Comment:
We can retain this JS Doc style comment coz it s still applicable to `generateSequence`